### PR TITLE
adjusted several texts (bsc#947276)

### PR DESCRIPTION
**Pull request by jsrain**

### CODE REVIEW

File: yast/src/include/smt/dialogs.rb [hunk 1]
```diff
@@ -1196,7 +1196,7 @@ def ValidateCredentialsDialog(id, event)
       if nuuser == nil || nuuser == ""
         UI.SetFocus(Id("NUUser"))
         # Pop-up error message
-        Report.Error(_("NU User must not be empty."))
+        Report.Error(_("Updatge server user must not be empty."))
```
Comment (mcalmer) [inline yast/src/include/smt/dialogs.rb]: Here is a typo

Comment (mcalmer) [inline yast/src/include/smt/dialogs.rb]: Hmm, maybe we use the term of SCC here? Organization User / Organization Password ?

File: yast/src/include/smt/wizard.rb [hunk 2]
```diff
@@ -260,7 +260,7 @@ def MainSequence(sequence_type)
 
       if sequence_type == "config"
         # TRANSLATORS: dialog caption
-        caption = _("NU Configuration")
+        caption = _("Update Server Configuration")
```
Comment (mcalmer) [inline yast/src/include/smt/wizard.rb]: Would be "Registration Server" better than "Update Server" ? (unsure)

Comment (jsrain) [inline yast/src/include/smt/wizard.rb]: No. Registration server is scc.suse.com, while update server is updates.suse.com, or?

File: yast/src/include/smt/dialogs.rb (hunk 1 above)
Comment (jsrain) [inline yast/src/include/smt/dialogs.rb]: Will fix the typo. Long-term, we should move towards SCC terms, but last time I checked even SCC was still somewhere between the two terminologies.

**Comment by mcalmer** [comment]
LGTM, feel free to merge

**Pull request merged**